packages/twenty-front/src/pages/settings/developers/playground/PlaygroundRouter.tsx

```diff
@@ -0,0 +1,20 @@
+import { GraphQLPlayground } from '@/settings/playground/components/GraphQLPlayground';
+import {
+  PlaygroundSchemas,
+  PlaygroundTypes,
+} from '@/settings/playground/components/PlaygroundSetupForm';
+import { RestPlayground } from '@/settings/playground/components/RestPlayground';
+import { useParams } from 'react-router-dom';
+
+export const PlaygroundRouter = () => {
+  const { schema, type } = useParams<{
+    schema: PlaygroundSchemas;
```

Comment:
I didn't manage to load the schema on the graphql console, did it work for you?

![Uploading Screenshot 2025-02-25 at 14.27.24.png…]()